src/core/useAtomValue.ts

```diff
@@ -0,0 +1,7 @@
+import { useAtom } from 'jotai'
+import type { Atom } from 'jotai'
+import type { Scope } from '../core/atom'
+
+export function useAtomValue<Value>(anAtom: Atom<Value>, scope?: Scope) {
+  return useAtom(anAtom, scope)[0]
```

Comment:
Let's move the implementation here.
In the end, our `useAtom` will be something like this (written without types to explain simply):
```js
export function useAtom(anAtom, scope) {
  return [useAtomValue(anAtom, scope), useSetAtom(anAtom, scope)]
}
```
The whole point of this change is to avoid duplicated code.